config/Zed/Oms/HeidelpaySubprocesses/Finalize.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<statemachine
+        xmlns="spryker:oms-01"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="spryker:oms-01 http://static.spryker.com/oms-01.xsd">
+
+    <process name="Finalize">
+        <states>
+            <state name="ready for finalize"/>
+            <state name="finalize request sent"/>
+            <state name="finalize succeeded"/>
+            <state name="invalid"/>
+        </states>
+
+        <transitions>
+            <!-- pre-finalize -->
+            <transition happy="true">
+                <source>ready for finalize</source>
```

Comment:
The noun is "finalization", or the sentence "ready to finalize".
but no big deal.